wrangler-core/src/main/java/io/cdap/wrangler/utils/PackagedSchemaLoader.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.wrangler.utils;
+
+import com.google.gson.Gson;
+import io.cdap.wrangler.utils.Manifest.Standard;
+import org.apache.avro.Schema;
+import org.apache.commons.collections4.SetValuedMap;
+import org.apache.commons.collections4.multimap.HashSetValuedHashMap;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Paths;
+import java.util.Map;
+
+/**
+ * The {@link PackagedSchemaLoader} object loads an AVRO data model schema bundled as a resource within a package.
+ */
+public class PackagedSchemaLoader implements AvroSchemaLoader {
```

Comment:
is this only used in tests?

Reply:
Removed and a stub for testing purposes has been added.